WooCommerce/src/main/kotlin/com/woocommerce/android/ui/products/inventory/ScanToUpdateInventoryViewModel.kt

```diff
@@ -95,6 +107,14 @@ class ScanToUpdateInventoryViewModel @Inject constructor(
     }
 
     private suspend fun handleProductNotFound(barcode: String) {
+        tracker.track(
+            AnalyticsEvent.PRODUCT_SEARCH_VIA_SKU_FAILURE,
+            mapOf(
+                AnalyticsTracker.SCAN_TO_UPDATE_INVENTORY to resourceProvider.getString(
+                    R.string.scan_to_update_inventory_unable_to_find_product, barcode
+                )
```

Comment:
```suggestion
                KEY_SCANNING_SOURCE to AnalyticsTracker.SCAN_TO_UPDATE_INVENTORY
```
The prop's key should be equal to "source", and value "scan_to_update_inventory"

Reply:
same as above https://github.com/woocommerce/woocommerce-android/pull/10339/commits/4ebb6f1ad3041450a1bd4394980692d0183d7928